test/python/circuit/test_parameters.py

```diff
@@ -424,7 +424,7 @@ def test_expressions_of_parameter_with_constant(self):
     def test_operating_on_a_parameter_with_a_non_float_will_raise(self):
         """Verify operations between a Parameter and a non-float will raise."""
 
-        bad_constants = [1j, '1', numpy.Inf, numpy.NaN, None, {}, []]
```

Comment:
Let's save complex support for another PR. There are other changes that would be needed.

Reply:
This PR originally had other changes to implement complex but I removed. Looks like I missed one. Thanks.